docs/how-to/code/internal-user/rockcraft.yaml

```diff
@@ -0,0 +1,32 @@
+name: internal-user
+base: bare
+build-base: ubuntu@24.04
+version: 'latest'
+summary: Add an internal user inside a rock
+description: Add an internal user inside a rock, and run a service with it.
+platforms:
+    amd64:
+
+# [docs:rock-services]
+services:
+    whoami:
+        override: replace
+        startup: enabled
+        command: bash -c "whoami; sleep 5"
```

Comment:
the sleep is an ugly trick in this case. I suggest running something useful. Can be something that sleeps, intentionally (rather than as a trick to avoid a bad exit code).

And then, as the last step in the doc (which is also way too cumbersome), you can do a `pebble exec --context` to run `whoami`

Reply:
The service was changed to a python http service.